src/MediaCodecList.h

```diff
@@ -26,13 +26,22 @@ class CJNIMediaCodecList : public CJNIBase
 {
 public:
   CJNIMediaCodecList(const jni::jhobject &object) : CJNIBase(object) {};
+  CJNIMediaCodecList(const int kind);
   //~CJNIMediaCodecList() {};
```

Comment:
please don't pass plain const into a function or return from a function. there's no benefit.
the commented destructor can be cleanup up
```suggestion
  CJNIMediaCodecList(int kind);
```

Reply:
ok, const is not necessary